py-pkg/uptasticsearch/clients.py

```diff
@@ -40,15 +40,13 @@ def search(self, body, index="", doc_type="", scroll_context_timer="1m", page_si
             body (json string): The query body
             index (string): The name of the Index to Query. Default: "" (no index)
             doc_type (string): The Doc Type to query. Default: "" (no doc_type)
-            scroll_context_timer (string): A string such as "1m" or "5m" that specifies
-                how long to keep the scroll context alive between pages for large
-                queries. Default: "1m"
+            scroll_context_timer (string): A string such as "1m" or "5m" that specifies how long to keep the scroll context alive between pages for large queries. Default: "1m"
```

Comment:
I _think_ Sphinx will recognize lines like this broken by explicit newlines, e.g. putting a `\n` at the end of each line.

But I have no strong preference between doing that and what you did here, so this is cool!